pkg/asset/quota/openstack/openstack.go

```diff
@@ -11,44 +14,79 @@ import (
 // These constraints can be used to check if there is enough quota for creating a cluster
 // for the install config.
 func Constraints(ci *validation.CloudInfo, controlPlanes []machineapi.Machine, computes []machineapi.MachineSet) []quota.Constraint {
-	ctrplConfigs := make([]*openstackprovider.OpenstackProviderSpec, len(controlPlanes))
-	for i, m := range controlPlanes {
-		ctrplConfigs[i] = m.Spec.ProviderSpec.Value.Object.(*openstackprovider.OpenstackProviderSpec)
+	var constraints []quota.Constraint
+
+	for i := 0; i < len(controlPlanes); i++ {
```

Comment:
```suggestion
	for _, controlPlaneMachine := range controlPlanes {
```
why not use the range here?